cypress/e2e/shared/buckets.test.ts

```diff
@@ -4,23 +4,13 @@ describe('Buckets', () => {
   beforeEach(() => {
     cy.flush()
 
-    cy.signin()
-      .then(() =>
-        cy.request({
-          method: 'GET',
-          url: '/api/v2/buckets',
+    cy.signin().then(() => {
```

Comment:
aside, but weird that this is in oss, cloud, and shared

Reply:
🤷‍♀️ Also the tests in the `cloud` folder are never run as part of your CI process. 
https://github.com/influxdata/ui/blob/e7e59e7/.circleci/config.yml#L195

Reply:
lol thanks for bringing that to my attention